packages/compartment-mapper/src/import-archive-lite.js

```diff
@@ -124,15 +124,12 @@ const makeArchiveImportHookMaker = (
             // note it's not being marked as exit in sources
             // it could get marked and the second pass, when the archive is being executed, would have the data
             // to enforce which exits can be dynamically imported
-            return {
-              record: await attenuateModuleHook(
-                moduleSpecifier,
-                record,
-                compartmentDescriptor.policy,
-                attenuators,
-              ),
-              specifier: moduleSpecifier,
-            };
+            return attenuateModuleHook(
```

Comment:
The scope of this PR has expanded beyond a `test:` now, right? Seems like a `feat:` or a `chore:` in preparation for a `feat:`.